snafu/flent_wrapper/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM registry.access.redhat.com/ubi8:latest
+
+RUN dnf install -y --nodocs git python3-pip && dnf clean all
+COPY /snafu/image_resources/centos8-appstream.repo /etc/yum.repos.d/centos8-appstream.repo
+RUN dnf install -y --nodocs redis --enablerepo=centos8-appstream && dnf clean all
+
+RUN yum -y install wget gcc make --nodocs
+RUN yum -y install bzip2 --nodocs
+RUN yum -y install iputils --nodocs
```

Comment:
Can you replace yum by dnf and group them?. 

On the other hand, I see you are building  iperf2 inside the container, we should use multistage builds to prevent having unnecessary packages in the container image, as we already do for FIO.

Reply:
It now uses a grouped dnf line, and it now uses multi-stage builds.